voyager/modules/client-bootstrap/movement/src/main.rs

```diff
@@ -99,6 +106,9 @@ pub struct Config {
 
     /// The RPC endpoint for custom movement apis.
     pub movement_rest_url: String,
+
+    /// The relayers that are allowed to modify this light client
+    pub whitelisted_relayers: Vec<cosmwasm_std::Addr>,
```

Comment:
lets put an explainer comment as to why this is a trusted light client for now

Reply:
correct